ROADMAP.md

```diff
@@ -123,21 +123,24 @@ Having the community guidelines pointed to, and creating custom ones as needed,
 
 ## Milestone - libp2p.io
 
-> Summary:
+> Summary: We need to ship a first version of the libp2p.io website
 
 ### Leads
 
-- David
-- Richard
+- @VictorBjelkholm
```

Comment:
Absolutely, I'll take ownership of the whole thing, making sure we have a first iteration of the website that is actually usable, which includes content :)